src/client/LocalServer.ts

```diff
@@ -17,6 +17,12 @@ import { defaultReplaySpeedMultiplier } from "./utilities/ReplaySpeedMultiplier"
 import { getPersistentID } from "./Main";
 import { z } from "zod";
 
+export class ReplayDataUpdateEvent {
```

Comment:
Let me know if there's a better place for this event. It seemed very specific to LocalServer, but I could see it being in its own file (maybe `utilities/`).